src/test/resources/edu/hm/hafner/analysis/parser/grype-report.json

```diff
@@ -0,0 +1,4002 @@
+{
```

Comment:
The file is quite big. Is there a way to reduce the number of lines without changing the test coverage?

Reply:
reduced size ofgrype-report.json to 24k